packages/frontend/src/pages/scaling-risk/view/ScalingRiskView.stories.tsx

```diff
@@ -20,6 +20,7 @@ export function RiskView() {
     <>
       <PageContent>
         <ScalingRiskView
+          upcomingEnabled={false}
```

Comment:
if you make this prop optional, you won't have to pass it. That would be the following of best practices.